core/src/main/scripts/importer/validateData.py

```diff
@@ -981,6 +981,22 @@ class MutationsExtendedValidator(Validator):
 
     NULL_AA_CHANGE_VALUES = ('', 'NULL', 'NA')
 
+    # MAF values for Variant_Classification column
+    # from https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+%28MAF%29+Specification + Unknown:
+    VARIANT_CLASSIFICATION_VALUES = [
+       'Frame_Shift_Del',
+       'Frame_Shift_Ins',
+       'In_Frame_Del',
+       'In_Frame_Ins',
+       'Missense_Mutation',
+       'Nonsense_Mutation',
+       'Splice_Site',
```

Comment:
While testing with SKCM TCGA, `Splice_Region` also occurred quite often:
`Splice_Region`: 2721
`Splice_Region`: 2895

Apparently it's unofficial, see https://github.com/mskcc/vcf2maf/issues/88. cBioPortal has no support for Splice_Region, so it's okay to not include it here.